src/cognito/post-confimation.js

```diff
@@ -0,0 +1,28 @@
+import {DynamoDB} from 'aws-sdk';
+import {CognitoIdentityServiceProvider} from 'aws-sdk';
+
+'use strict';
+import * as crypto from 'crypto';
+
+let cognitoidentityserviceprovider = new CognitoIdentityServiceProvider();
+let dynamoDB = new DynamoDB.DocumentClient();
```

Comment:
Always try to use const if the value doesn't change. [read this](https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75)